octomap:
    pointcloud_topic: "/semantic_pcl/semantic_pcl"
    tree_type: 1
    world_frame_id: "/map"
    resolution: 0.1
    max_range: 3.0
    raycast_range: 2.0
    clamping_thres_min: 0.2
    clamping_thres_max: 0.9
    occupancy_thres: 0.5
    prob_hit: 0.6
    prob_miss: 0.4
    save_path : "~/map.bt"
